certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -61,28 +61,20 @@ def test_manual_dns_auth(context):
     assert_save_renew_hook(context.config_dir, certname)
 
 
-def test_renew(context):
-    """Test various certificate renew scenarios."""
-    # First, we create a target certificate, with all hook dirs instantiated.
-    # We should have a new certificate, with hooks executed.
-    # Check also file permissions.
+def test_renew_files_permissions(context):
+    """Test certificate file permissions upon renewal"""
     certname = context.get_domain('renew')
-    context.certbot([
-        'certonly', '-d', certname, '--rsa-key-size', '4096',
-        '--preferred-challenges', 'http-01'
-    ])
+    context.certbot(['-d', certname])
 
     assert_certs_count_for_lineage(context.config_dir, certname, 1)
     assert_world_permissions(
         join(context.config_dir, 'archive/{0}/privkey1.pem'.format(certname)), 0)
 
-    # Second, we force renew, and ensure that renewal hooks files are executed.
-    # Also check that file permissions are correct.
-    misc.generate_test_file_hooks(context.config_dir, context.hook_probe)
+    # Force renew. Assert certificate renewal and proper permissions.
+    # We assert hook scripts execution, certificate renewal and proper permissions.
```

Comment:
We no longer assert hook scripts execution here.

Reply:
Comment corrected.